### PR TITLE
SQS Consumer upgrades to Spring Cloud. 

### PR DESCRIPTION
**Pull request by atharmajeed**
## What

Link to story

https://dsdmoj.atlassian.net/browse/LASB-939 
https://dsdmoj.atlassian.net/browse/LASB-2066

Please, don't merge 

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: maat-court-data-api/src/main/java/gov/uk/courtdata/link/service/CreateLinkCpJobStatusListener.java [hunk 1]
```diff
@@ -23,7 +24,9 @@ public class CreateLinkCpJobStatusListener {
     private final Gson gson;
     private final QueueMessageLogService queueMessageLogService;
 
-    @JmsListener(destination = "${cloud-platform.aws.sqs.queue.createLinkCpStatusJob}")
+//This queue is not in use anymore - May 2023
+@SqsListener(value = "${cloud-platform.aws.sqs.queue.createLinkCpStatusJob}",
```
Comment (nittagr) [inline maat-court-data-api/src/main/java/gov/uk/courtdata/link/service/CreateLinkCpJobStatusListener.java:28]: Can we delete this if it is not in use? please verify.

**Comment by nittagr** [comment]
Please resolve conflicts, thanks

File: maat-court-data-api/src/main/java/gov/uk/courtdata/link/service/CreateLinkCpJobStatusListener.java (hunk 1 above)
Comment (atharmajeed) [reply maat-court-data-api/src/main/java/gov/uk/courtdata/link/service/CreateLinkCpJobStatusListener.java:28]: We need to double-check this and if we decided to remove then need to update the terraform script on the cloud platform,

 however, I am going to leave this for now.

Comment (atharmajeed) [reply maat-court-data-api/src/main/java/gov/uk/courtdata/link/service/CreateLinkCpJobStatusListener.java:28]: So mark this as resolved.

**Comment by atharmajeed** [comment]
I've updated the PR and please review.

**Review by nittagr — APPROVED** [review]

**Pull request merged**